.github/workflows/build.yml

```diff
@@ -20,18 +20,18 @@ jobs:
         uses: docker/metadata-action@v5.5.1
         with:
           images: |
-            ghcr.io/A2va/file-transfer
+            ghcr.io/A2va/f-transfer
```

Comment:
There would have been no need to use search and replace x)

Reply:
...